README.md

```diff
@@ -21,22 +21,26 @@ The example below initializes a web project with `launchpad init`, and deploys t
 
 ## Benefits
 
-### Build publish and deploy with a single command
+### Build, publish, and deploy with a single command
 
-Launchpad can build any image, publish it to your Docker registry, and deploy it to Kubernetes in one step.
+No need to remember which container image you uploaded to where, and what kube-context to use, Launchpad can build any image, publish it to your Docker registry, and deploy it to Kubernetes in one step.
 
-### A consistent deployment setup per project
 
-Launchpad defines your project's Kubernetes configuration. Whether you are deploying in development, staging, or production environment, whether you are deploying to a local or remote Kubernetes cluster, Launchpad always creates the same set of Kubernetes resources for you each time.
+### A Heroku-like experience, except on your Kubernetes cluster
```

Comment:
Lol, when I first read this I thought it meant "A Heroku like Experience [but not] on your Kubernetes cluster". Let's do: 

> A Heroku-like exprience on your own Kubernetes cluster